sw/device/tests/closed_source/meson.build

```diff
@@ -0,0 +1,37 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# All build targets below will be build in closed source envirounment only. 
+# In order to build these targets, 'build_closed_source' and 'closed_source_dir'
+# compilation flags should be defined.
```

Comment:
nit: "compilation" --> "build" ; they aren't really compilation flags per se, since meson is a build system not a compiler